db/seeds.rb

```diff
@@ -2,16 +2,15 @@
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
 
-admin = User.create!(username:"admin")
 
 10.times do
     Item.create!(
         name: Faker::Address.street_name,
-        description: Faker::Lorem.sentence(word_count: 15), 
+        description: Faker::Lorem.sentence(word_count: 5), 
         image: Faker::Placeholdit.image(size: '350x250', format: 'jpg'), 
         price: rand(1000..10000), 
         city: Faker::Address.city, 
         capacity: rand(500..5000), 
-        user_id: admin.id
+        user: User.first
```

Comment:
After we deleted line 5 which is creating the first user,

what will be the return of line 14 ? `User.first`

Reply:
I have @mckent05 permission to add the line again :+1: